### ISSUE TITLE
about classifier weight initial

### ISSUE THREAD

**Issue by Youskrpig** [yxgeee/MMT#36]
Hi, thanks for your great work, i noticed in base_train_dbscan.py, line 171  
model.module.classifier.weight.data[:args.num_clusters].copy_(F.normalize(cluster_centers, dim=1).float().cuda())
what does it mean and why not directly setting classifier weight==0?
Looking for your reply.

**Comment by yxgeee** [comment]
We use cluster centroids to initialize the classifier weights for better convergence.

**Comment by Youskrpig** [comment]
get it
